SU-SWS/decanter
# It is good to publish things when you change versions.

# READY FOR REVIEW

# Summary
- I failed to run `npm run publish` after updating the package files and the build failed. Included in this is a fix for 6.0.2 to include the new stuff.

# Needed By (Date)
- ASAP

# Urgency
- HIGH - 6.0.2 fails

# Steps to Test

1. Check out this branch
2. Run `nvm use` && `npm install` && `npm run publish`
3. Validate no more changes and build works

# Affected Projects or Products
- Decanter et all
